maven/build.xml

```diff
@@ -27,12 +27,12 @@
 <!-- ================================================= -->
 <!--            Maven Bundle Buildfile                 -->
 <!-- ================================================= -->
-<project name="jython-maven" default="bundle-all" basedir="..">
+<project name="jython-maven" default="bundle-all" basedir=".." xmlns:artifact="antlib:org.apache.maven.artifact.ant">
 
   <property file="${user.home}/ant.properties" />
   <property file="${basedir}/default.properties"/>
 
-  <property name="project.version" value="2.7.0-SNAPSHOT"/>
+  <property name="project.version" value="2.7.1-rc10"/>
```

Comment:
Fair enough on the version name. 

This will be of the path ```com.datastax.opscenter.jython-standalone```. Do we want another ```jython``` in there?